src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,48 @@
+.watch {
+  @include circle($widthClock);
+
+  border: 1vmin dotted $colorBorder;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &::before {
+    @include after-before($widthSecond, $heightSecond, $colorSecond, 2);
+
+    animation-duration: $stepSecond;
+    animation-timing-function: linear;
+  }
+
+  &::after {
+    @include after-before($widthMinute, $heightMinute, $colorMinute, 1);
+
+    animation-duration: $stepSecond * 60;
+    animation-timing-function: steps(60, end);
+  }
+
+  &__center {
+    @include circle($centerClock);
+
+    background-color: $colorCenter;
+    position: relative;
+    z-index: 3;
+  }
+
+  &--speed-up {
+    &::before {
+      animation-duration: $speedUp;
+    }
+    &::after {
+      animation-duration: $speedUp * 60;
+    }
+  }
+}
+
+@keyframes clockwork {
+  from {
+    transform: translateX(-50%) rotate(0deg);
+  }
+  to {
+    transform: translateX(-50%) rotate(360deg);
+  }
+}
```

Comment:
The `@keyframes` rule is well implemented, but it would be better if you add a comment to explain what the animation is doing. Remember, comments are an important part of code readability and maintainability.